From: Wang Sheng-Hui <shhuiw@gmail.com>
Date: Thu, 8 Sep 2011 12:32:14 +0200
Subject: elevator: use ELV_NAME_MAX instead of magic number 16 for
Git-commit: 484fc254b88257a2d8b3759aa062e8e8b35e0988
References: bnc#778477
Patch-Mainline: v3.2
 chosen_elevator

We have ELV_NAME_MAX defined to 16, and hence we should use it
instead of the magic nubmer 16 for elevator's name string.

Signed-off-by: Wang Sheng-Hui <shhuiw@gmail.com>
Signed-off-by: Jens Axboe <jaxboe@fusionio.com>
Acked-by: Hannes Reinecke <hare@suse.de>
---
 block/elevator.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/elevator.c b/block/elevator.c
index 01ee722..fa63bfa 100644
--- a/block/elevator.c
+++ b/block/elevator.c
@@ -182,7 +182,7 @@ static void elevator_attach(struct request_queue *q, struct elevator_queue *eq,
 	eq->elevator_data = data;
 }
 
-static char chosen_elevator[16];
+static char chosen_elevator[ELV_NAME_MAX];
 
 static int __init elevator_setup(char *str)
 {
-- 
1.7.10.4

